### ISSUE TITLE
Document the purpose of each open port

### ISSUE THREAD

**Issue by lbeltrame** [horahoradev/horahora#61]
This is important for those who run this service along others (potential port conflicts).  In particular, what needs to be known if any of these need to be actually exposed. Is the webapp required to listen on port 3000 outside the container, for example?

**Comment by GabenGar** [comment]
Yes and no. In a proper setup there shouldn't be `webapp` in production, but in development it is required for dev QoL to work.

**Comment by horahoradev** [comment]
Sure, I'll add this to my to-do list; thanks for the suggestion.
